lib/tropic/tropic_paragraph.flow

```diff
@@ -944,23 +948,31 @@ commonWordConstructor(
 	);
 }
 
+letterSpacingSuportedSizeInspector(
+	inspector : ParaElementInspector,
+	letterSpacing : double,
+	view : Tropic
+) -> Tropic {
+	sizeB = make(zeroWH);
+	uns = subscribe2(sizeB, \v -> {
```

Comment:
it should be in the constuctor, not outside

Reply:
But it's you made it so in the first place. I guess, there was some reason for that.
https://github.com/area9innovation/flow9/commit/0e7407b4e288be0a359e5ca1921f7bc28fd75438#diff-88d1bfbd82d2dc7b7d301af30d100d3d2c133e53363775e8dd7de2be6be413a5R838